app/controllers/courses_controller.rb

```diff
@@ -1,5 +1,5 @@
 class CoursesController < ApplicationController
   def index
-    # @courses = current_user.courses
+    @courses = current_user.courses.includes(:roles)
```

Comment:
I made the changes in my `dev` branch for this here, but I still feel a bit off about the logic now. If we iterate through the roles, get the `Course` associated with each `Role` object -- what do we do in the case of multiple roles for the same user for a particular course?

**Example**: Some user is an *Instructor* and a *Student* for a course.
**Question**: Is this a problem? Will a user have multiple roles in a course?